LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-event-handlers-content-attributes.tentative-expected.txt

```diff
@@ -455,4 +455,10 @@ PASS AUDIO.setAttribute(onwaitingforkey, "unsafe_handler()") calls default polic
 PASS AUDIO.setAttributeNS(onwaitingforkey, "unsafe_handler()") calls default policy
 PASS VIDEO.setAttribute(onwaitingforkey, "unsafe_handler()") calls default policy
 PASS VIDEO.setAttributeNS(onwaitingforkey, "unsafe_handler()") calls default policy
+PASS DIV.setAttribute("onafterprint", "unsafe_handler()") calls default policy
+PASS DIV.setAttribute("onwaitingforkey", "unsafe_handler()") calls default policy
+FAIL DIV.setAttribute("onbegin", "unsafe_handler()") calls default policy assert_equals: expected (string) "Element onbegin" but got (undefined) undefined
```

Comment:
See https://bugs.webkit.org/show_bug.cgi?id=286916 for bug to track fixing onbegin.

Reply:
Turns out these event handlers actually aren't supported in WebKit yet so it's not a current issue. When they're added care will neeed to be taken that this is addressed.